compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -16330,6 +16331,7 @@ private STNode parseTypedBindingPatternTypeRhs(STNode typeDesc, ParserRuleContex
     private STNode parseTypedBindingPatternTypeRhs(STNode typeDesc, ParserRuleContext context, boolean isRoot) {
         STToken nextToken = peek();
         switch (nextToken.kind) {
+            case SEMICOLON_TOKEN:
```

Comment:
Didn't get the reason why we need semicolon here. Do we have a sample?

Reply:
Removed. 95cca811288f371b680fde4962089b1eb94ab90c